src/core/icons/linkedin.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+export const SvgLinkedin = (): JSX.Element => (
+	<svg
+		width="30"
+		height="27"
+		viewBox="0 0 30 27"
```

Comment:
Oof, the viewbox is sized incorrectly, it should be 30x30. We'll fix that in Figma, sorry!